src/components/useCountries.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './useCountries.css';
+
+export default function CountryDisplay({ name, iso2 }) {
```

Comment:
It's good practice to have the function name match the component name. Calling this component useCountries is confusing, because you already have a useCountries in your hooks folder, and the word 'use' for naming is almost exclusively used for hooks. I'd strongly reccomend changing the name of this component.